src/main/java/uk/gov/companieshouse/model/IllnessReason.java

```diff
@@ -50,4 +69,16 @@ public String getIllnessImpactFurtherInformation() {
     public List<Attachment> getAttachments() {
         return attachments;
     }
+
+    public void setAttachments(final List<Attachment> attachments) {
+        if (attachments == null) {
+            this.attachments = Collections.emptyList();
+        }
```

Comment:
Collections.emptyList() returns an immutable list. Doing this might prevent any attachments from being added.
https://www.tutorialspoint.com/java/util/collections_emptylist.htm

Thats just a general tip for the future, the important thing at the moment here is that it is immediately overwritten by the parameter so this can just be removed.

Reply:
This can still be removed.

Reply:
Done